vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java

```diff
@@ -467,25 +467,25 @@ else if (!target.targetType.isCurrent() && !fastMap.isEmpty() && !fastMap.equals
                   switch (target.targetType) {
                   case ZONE:
                   case CURZONE:
-                    if (!fastZone.equals(gamePiece.getProperty(BasicPiece.CURRENT_ZONE))) {
+                    if ((fastZone != null) && !fastZone.equals(gamePiece.getProperty(BasicPiece.CURRENT_ZONE))) {
```

Comment:
Should this to be `fastZone == null ||` instead? If I understand this right, the `continue` case is for a non-match.

Reply:
done